webapp/src/components/expanded_view/component.tsx

```diff
@@ -702,6 +728,24 @@ export default class ExpandedView extends React.PureComponent<Props, State> {
                                 }
                                 unavailable={noInputDevices || noAudioPermissions}
                             />
+                            {globalRhsSupported && (
+                                <ControlsButton
+                                    id='calls-popout-chat-button'
+                                    onToggle={this.toggleChat}
+                                    tooltipText={chatToolTipText}
+                                    tooltipSubtext={chatToolTipSubtext}
+                                    // eslint-disable-next-line no-undefined
+                                    shortcut={undefined}
+                                    bgColor={''}
+                                    icon={
+                                        <ProductChannelsIcon // TODO use 'icon-message-text-outline' once added
+                                            size={28}
+                                            color={'white'}
+                                        />
+                                    }
+                                    unavailable={noInputDevices || noAudioPermissions}
```

Comment:
We should remove this as it doesn't apply to the chat thread button.

Reply:
Condition updated for the case of `currentTeam` != call channe team